examples/deepspeech/README.md

```diff
@@ -4,5 +4,7 @@ The distributed DeepSpeech training application that had been used as a test
 application and requirement provider towards TensorHive is now in a
 [separate repository](https://github.com/roscisz/dnn_training_benchmarks/tree/master/TensorFlowV1_DeepSpeech_ldc93s1).
 
-See the [TensorHive examples directory](https://github.com/roscisz/TensorHive/tree/master/examples) for
-examples how TensorHive task execution module can be used for various training applications. 
\ No newline at end of file
+The application has been also used in the usage example of the TensorHive task execution
+module for the [TensorFlow ClusterSpec](https://github.com/roscisz/TensorHive/tree/master/examples/TensorFlow_ClusterSpec)
+scenario. See also the [TensorHive examples directory](https://github.com/roscisz/TensorHive/tree/master/examples) for other
+examples how TensorHive task execution module can be used for various training scenarios. 
```

Comment:
Sounds a bit awkward to me, maybe something like:
> The application has been also used in the TensorHive task execution
> module example for the [TensorFlow ClusterSpec](https://github.com/roscisz/TensorHive/tree/master/examples/TensorFlow_ClusterSpec)
> scenario. See also the [TensorHive examples directory](https://github.com/roscisz/TensorHive/tree/master/examples) for other
> examples how TensorHive task execution module can be used for various training scenarios.

Reply:
fixed in b42594add4945f72a9a9a97d703699c179a3de39